arxiv/marxdown/tests/test_serve.py

```diff
@@ -202,3 +210,33 @@ def test_serve_with_htm(self):
                              status.HTTP_302_FOUND)
             self.assertEqual(response.headers['Location'],
                              'http://localhost/baz/redirectme')
+
+    @mock.patch(f'{factory.__name__}.config', CONFIG)
+    def test_serve_static(self):
+        """Requests for static URLs should be redirected."""
+        app = factory.create_web_app()
+        client = app.test_client()
+
+        with app.app_context():
+            response = client.get('/notapage.txt', follow_redirects=False)
+            self.assertEqual(response.status_code, status.HTTP_302_FOUND)
+            self.assertEqual(
+                response.headers['Location'],
+                "https://%s.s3.amazonaws.com/static/arxiv.marxdown/%s/%s/%s"
+                % (S3_BUCKET, VERSION, SITE_NAME, "notapage.txt")
+            )
+
+    @mock.patch(f'{factory.__name__}.config', CONFIG_WITH_CDN)
+    def test_serve_cdn_static(self):
```

Comment:
Thanks, this will make it clearer that CDN config is an option for those not familiar with `FLASKS3`.